build.sc

```diff
@@ -416,6 +422,38 @@ trait ScalaParse extends SbtModule with ScalaCliPublishModule with settings.Scal
   def scalaVersion = Scala.defaultInternal
 }
 
+trait Scala3Runtime extends SbtModule with ScalaCliPublishModule with settings.ScalaCliCompile {
+  def ivyDeps      = super.ivyDeps() ++ Agg(Deps.scalaparse)
+  def scalaVersion = Scala.scala3
+  def jar = T {
+    val original = super.jar().path
+    // scala3RuntimeFixes.jar is also used within
+    // resource-config.json and BytecodeProcessor.scala
+    val dest = original / os.up / "scala3RuntimeFixes.jar"
+    os.copy(original, dest)
+    PathRef(dest)
+  }
```

Comment:
I'm not sure that's the right place to do this. If this file needs to be added under a specific name as resource, this can be handled in the module that adds it as a resource.
```suggestion
```